src/DIRAC/Resources/Catalog/RucioFileCatalogClient.py

```diff
@@ -697,3 +697,95 @@ def getDirectorySize(self, lfns, longOutput=False, rawFiles=False):
             except Exception as err:
                 return S_ERROR(str(err))
         return S_OK(resDict)
+
+    @checkCatalogArguments
+    def getFileUserMetadata(self, path):
+        """Get the meta data attached to a file, but also to
+        all its parents
+        """
```

Comment:
Describe method arguments for all methods (and their return objects)

Reply:
Will do